src/server/formations-initiales-detail/domain/formationInitiale.ts

```diff
@@ -0,0 +1,5 @@
+import { FormationInitialeDetailCMS } from '~/server/cms/domain/formationInitiale.type';
+import { FormationInitialeDetail } from '~/server/formations-initiales/domain/formationInitiale';
+
+export type FormationInitialeDetailComplete =  FormationInitialeDetail & FormationInitialeDetailCMS;
```

Comment:
Est-ce qu'on ne peut pas réutiliser l'interface `FormationInitiale`  ?

```suggestion
export type FormationInitialeAvecDetails =  FormationInitiale & DetailsFormationInitiale;
```